sssd_test_framework/hosts/client.py

```diff
@@ -149,13 +149,13 @@ def restore(self, backup_data: Any | None) -> None:
                 fi
             }}
 
+            cp --force --archive "{backup_path}/home/*" /home/ || :
             rm --force --recursive /etc/sssd /var/lib/sss /var/log/sssd /home/*
```

Comment:
you are removing /home/* here

Reply:
Nice catch, it needs to be moved after that line.